generated/java/google-longrunning-v1/src/main/java/com/google/longrunning/OperationsClient.java

```diff
@@ -198,16 +201,17 @@ public final OperationsSettings getSettings() {
    * <p>Sample code:
    *
    * <pre><code>
-   * try (OperationsApi operationsApi = OperationsApi.create()) {
-   *   String formattedName = OperationsApi.formatOperationPathName("[OPERATION_PATH]");
-   *   Operation response = operationsApi.getOperation(formattedName);
+   * try (OperationsClient operationsClient = OperationsClient.create()) {
+   *   String formattedName = OperationsClient.formatOperationPathName("[OPERATION_PATH]");
+   *   Operation response = operationsClient.getOperation(formattedName);
    * }
    * </code></pre>
    *
    * @param name The name of the operation resource.
    * @throws com.google.api.gax.grpc.ApiException if the remote call fails
    */
   public final Operation getOperation(String name) {
+    OPERATION_PATH_PATH_TEMPLATE.validate(name, "getOperation");
```

Comment:
Is it the case that the operation name must always be formatted this way? The comment in the proto seems to hedge a bit: https://github.com/googleapis/googleapis/blob/master/google/longrunning/operations.proto#L84